lua/customroles/renegade.lua

```diff
@@ -24,6 +24,7 @@ it's completely up to you!
 
 These are the {traitors}:
 {traitorlist}]]
+ROLE.shortdesc = "Can see and be seen by the traitors as well as use their secret traitor chat. Can work with the traitors or against them. Wins by being the last player alive."
```

Comment:
```suggestion
ROLE.shortdesc = "Can see and be seen by the traitors and so must choose to work with or against them. Wins by being the last player alive."
```
I would cut the traitor chat bit because I feel like it is getting a bit too into specifics